ecommerce_connector/models/ecommerce_connection.py

```diff
@@ -53,6 +53,10 @@ def _selection_languages(self):
         default="ecommerce_id",
         required=True
     )
+    contact_search_partner_type = fields.Boolean(
```

Comment:
Couldn't it be true by default?

Reply:
I would't set it True by default as it might duplicate contacts.